db/schema.rb

```diff
@@ -213,7 +213,7 @@
     t.boolean "is_active", null: false
     t.integer "created_by", null: false
     t.datetime "created_at", null: false
-    t.integer "revenue_account", null: false
+    t.string "revenue_account", null: false
```

Comment:
@jossim this doesn't look right.  I think your local db might be in a funky state. See https://github.com/tablexi/nucore-open/pull/3137